src/DIPS.Xamarin.UI/Controls/Toast/ToastView.xaml.cs

```diff
@@ -0,0 +1,218 @@
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace DIPS.Xamarin.UI.Controls.Toast
+{
```

Comment:
From my understanding, this will never be directly used in XAML for the consumer. If so, move this to the internal/Xaml namespace to prevent it from popping up as a Xaml choice for our consumers.

See SheetView if you want an example

Reply:
Done